frontend/src/util/api.ts

```diff
@@ -71,16 +71,23 @@ export async function updateQuestion(
   questionId: number,
   newQuestion: NewQuestion | Question,
   token: string
-): Promise<Response> {
+): Promise<[Response, Response]> {
   const url: string = getBaseUrl();
+  const questionOnly = Object.assign(
+    {},
+    { QuestionOptions: undefined },
+    newQuestion
+  );
 
-  const response: Response = await sendPut(
-    `${url}/question/${questionId}`,
-    newQuestion,
+  const qres = sendPut(`${url}/question/${questionId}`, questionOnly, token);
+
+  const qores = sendPut(
```

Comment:
Can you add types for the variables in this function? So `qres` and `qores` are both `Response`. Not sure what exactly `questionOnly` would be... idk. Might need to make a new type, probably doesn't matter too much.

More descriptive response names would be better IMO but that's not too important, don't consider that part of the requested changes.

Reply:
ill fix this section to have more context

Reply:
![Screen Shot 2021-04-20 at 7 30 03 PM](https://user-images.githubusercontent.com/34965892/115475977-ceefdc80-a20e-11eb-9c94-2cc901aea434.png)

I don't think I need to add types to those variables since VS Code knows the type thanks to the function `sendPut`

ill add it here just so the syntax follows the styles of the file